cookbooks/aws-parallelcluster-environment/spec/unit/recipes/imds_spec.rb

```diff
@@ -56,11 +56,6 @@
           is_expected.to run_execute("Save ip6tables rules").with(command: /ip6tables-save/)
         end
 
-        it 'creates iptables init.d file' do
```

Comment:
Different templates are created as part of this PR (eg: parallelcluster-iptables.service), so it should be covered in this test.

Reply:
Done